security/certificates/updating-ca-bundle.adoc

```diff
@@ -6,6 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
+====
+Updating the certificate authority (CA) might cause the nodes of your cluster to reboot.
```

Comment:
Instead of "might cause", per https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/proxy-certificates.html#customization which says "the node is rebooted", seems it will definitely cause reboot. So "might" is not precise? (Just personal opinion. The Networking QE @zhaozhanqi can confirm)

Reply:
I have no idea this, I did not try to update the CA.  @lihongan @anuragthehatter if you know that.